src/gen/pdb2pqr.py

```diff
@@ -0,0 +1,123 @@
+        ]
+    console.print("Please choose a forcefield to use:")
+    # Choose one item from a list
+    forcefield = select(forcefields, cursor="🢧", cursor_style="cyan")
+    #making the cl prompt
+    if forcefield != "[red]User-defined Forcefield[/red]":
+        forcefield_str = f"--ff={forcefield}"
+        result.append(forcefield_str)
+        
+    else:
+        pass #still have to do something
+    
+    
+    
+    #output naming scheme options
+    naming_schemes = [
+            "AMBER",
+            "SCHARM",
+            "PEOEPB",
+            "PARSE",
+            "SWANSON",
+            "TYLO6",
+            "[red]Internal naming scheme[/red]"
+        ]
+    console.print("Please choose an output naming scheme to use:")
+    # Choose one item from a list
+    naming_scheme = select(naming_schemes, cursor="🢧", cursor_style="cyan")
+    #making the cl prompt
+    if naming_scheme != "[red]Internal naming scheme[/red]":
+        naming_scheme_str = f"--ffout={naming_scheme}"
+        result.append(naming_scheme_str)
+    
+    #If PARSE is the forcefield
+    if forcefield == "PARSE":
+        console.print("Options required for PARSE Forcefield")
+        parse_options = {"Make the protein's N-terminus neutral" : "--neutraln",
+                "Make the protein's C-terminus neutral" : "--neutralc",
+                }
+    
+        # Choose multiple options from a list
+        items1 = select_multiple(list(parse_options.keys()), tick_character='🎒', ticked_indices=[0], maximal_count=len(parse_options))
+    
+        for key, value in parse_options.items():
+            if key in items1:
+                result.append(value)
+    
+    #Additional Options
+    console.print("Additional Options")
+    
+    add_options = ["Ensure that new atoms are not rebuilt too close to existing atoms",
+                   "Optimize the hydrogen bonding network",
+                   "Assign charges to the ligand specified in a MOL2 file",
+                   "Enter the name for your APBS input file",
+                   "Add/keep chain IDs in the PQR file",
+                   "Insert whitespaces between atom name and residue name, between x and y, and between y and z",
+                   "Remove the waters from the output file"]
+    
+    values = ["--nodebump",
+              "--noopt",
+              "",
+              "",
+              "--keep-chain",
+              "--whitespace",
+              "--drop-water"]
+    
+    # Choose multiple options from a list
+    items2 = select_multiple(add_options, tick_character='*', ticked_indices=[0], maximal_count=len(add_options))
+    
+    for i in range(len(add_options)):
+        if i <= 1:
+            if add_options[i] not in items2:
+                result.append(values[i])
+        elif 2 <= i <= 3:
+            #CODE TO GET INPUT FILES FROM USER
+            pass
+        else: 
+            if add_options[i] in items2:
+                result.append(values[i])
+    
```

Comment:
I am inputting the pdb file path. Now set this part of the code as you have thought about.

Reply:
All the parts of the code that are yet to be added revolve around allowing the user to upload files. I'll figure out how to do that tomorrow.
I guess I can do it with the requests module.

Reply:
Here is how i would like you to do it, make a seperate function which you ask for if you have the pdb downloaded or you want to "request" and download it(as you will do it).
Now if he has the file downloaded, take the file and check its existance(make sure absolute and relative both work). Like i dont want to enter /home/some/path/7y6i.pdb if i am in the same path.

Now input the file path of pdb to this existing functoin and inside the `elif 2<=i <=3` code, add the file path as args. Hope thats clear

Reply:
I've made a separate function, check if this is how you envisioned it. I'm not sure how I'm supposed to 'check the existance of the path'